backend/pkg/modules/internal/core/core.go

```diff
@@ -86,7 +86,9 @@ func (c *core) EventNotify(ctx context.Context, event *Event) {
 func (c *core) HTTPHandler() http.Handler {
 	handler := http.NewServeMux()
 	for _, m := range c.modules {
-		handler.Handle(BaseHTTPPath+"/"+m.Name()+"/", m.HTTPHandler())
+		if m.HTTPHandler() != nil {
```

Comment:
let's create a dummy handler with stop and start and implement it once selective traces will be available (and remove this check)